tools/aws_lab_setup/sample-vars.yml

```diff
@@ -0,0 +1,15 @@
+ec2_region: us-east-1                 # region where the nodes will live
+ec2_az: us-east-1a                    # availability zone
+ec2_name_prefix: SEANTEST1120         # name prefix for all the VMs
+admin_password: ansible
+## Optional Variables
+#ec2_vpc_id: vpc-1234aaaa              # EC2 VPC ID in your region, if not supplied will be automatically created
```

Comment:
This is the next variable (vpc_subnet_id) should be required.

Reply:
this also inherits from the networking workshop, but I left it like this because I am not sure if you want people changing the subnet on the essentials side.

Reply:
The subnet ID is tied to a specific AWS account and VPC in that so yes. Eventually we want all of that to go away, but that's not here yet.

Reply:
So based in [my previous comment](https://github.com/ansible/lightbulb/pull/249#discussion_r163935335) it looks like you build in the tasks to create this so explicitly defining the subnet and vpc is not necessary and really doesn't need to be an option.

Reply:
ok lol can i remove it...

Reply:
sorry this was a question, can I remove (and/or deprecate) manually setting VPCs, subnets, etc.

Reply:
As mentioned [above](https://github.com/ansible/lightbulb/pull/249#discussion_r164902171) I'm fine with deprecating.

Reply:
@tima I can do that in another PR, the point of this PR was not to deprecate anything.  For now I have removed the options from the sample-vars for networking